libpius/signer.py

```diff
@@ -618,6 +619,9 @@ def sign_uid(self, key, index, level):
         print '  UID already signed'
         gpg.stdin.write('quit\n')
         return False
+      elif (PiusSigner.GPG_KEY_CONSIDERED in line):
+        debug("Got KEY_CONSIDERED")
+        continue
```

Comment:
This very same check is also necessary in method encrypt_signed_uid (line 391), otherwise I still get
  File "./pius", line 341, in <module>
    main()
  File "./pius", line 325, in main
    if signer.sign_all_uids(key, retval):
  File "/home/eskultety/git/pius/libpius/signer.py", line 777, in sign_all_uids
    self.encrypt_signed_uid(key, uid['file'])
  File "/home/eskultety/git/pius/libpius/signer.py", line 440, in encrypt_signed_uid
    raise EncryptionUnknownError(line)
libpius.exceptions.EncryptionUnknownError: [GNUPG:] KEY_CONSIDERED